whats-new-panel.yaml

```diff
@@ -257,3 +257,23 @@
   template: whatsnew_panel_message
   trigger:
     id: whatsNewPanelOpened
+- id: WHATS_NEW_AWESOMEBAR_74
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_AWESOMEBAR_74
+    published_date: 1582219877400
+    title:
+      string_id: cfr-whatsnew-searchbar-title
+    icon_url: data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxkZWZzPjxwYXRoIGQ9Ik0xNS43NDUgMTQuMzMxbC00LjgyMi00LjgyMmE2LjAxOSA2LjAxOSAwIDEgMC0xLjQxNCAxLjQxNGw0LjgyMiA0LjgyMmExIDEgMCAwIDAgMS40MTQtMS40MTR6bS05LjcwNy00LjI5M2E0IDQgMCAxIDEgMC04IDQgNCAwIDAgMSAwIDh6IiBpZD0iYSIvPjwvZGVmcz48ZyBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPjxyZWN0IGZpbGw9IiMwMDYwREYiIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgcng9IjIiLz48ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSg4IDgpIj48cGF0aCBkPSJNMCAwaDE2djE2SDB6Ii8+PG1hc2sgaWQ9ImIiIGZpbGw9IiNmZmYiPjx1c2UgeGxpbms6aHJlZj0iI2EiLz48L21hc2s+PHBhdGggZD0iTTAgMGgxNnYxNkgweiIgbWFzaz0idXJsKCNiKSIgZmlsbD0iI0ZGRiIvPjwvZz48L2c+PC9zdmc+,
+    icon_alt:
+      string_id: cfr-whatsnew-searchbar-icon-alt-text
+    body:
+      string_id: cfr-whatsnew-searchbar-body-enginename,
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/address-bar-search
+    cta_type: OPEN_URL,
+    link_text:
+      string_id: cfr-whatsnew-pip-cta,
```

Comment:
Just to confirm, are we going to reuse the PIP link_text here?

Reply:
Yes.